packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -150,6 +158,11 @@ export default class DataGrid extends React.Component {
 	}
 
 	getAgGridConfig() {
+		let rowData = this.props.rowData;
```

Comment:
This props is used ? If yes, could you add the props on the proptypes ?

Reply:
It's done :)